Source/Renamer/KerbalRenamer.cs

```diff
@@ -42,13 +42,13 @@ public class KerbalRenamer : MonoBehaviour
         public static KerbalRenamer rInstance = null;
         public string cultureDescriptor = "Culture";
         
-        // Holds the cultures (as instances) loaded from the KERBALRENAMER node(s)
+        /// Holds the cultures (as instances) loaded from the KERBALRENAMER node(s)
```

Comment:
Those should be in format 
```
        /// <summary>
        /// TODO
        /// </summary>

Reply:
Reverted to LogUtils as suggested. Added proper format for attributes both in KerbalRenamer.cs and Culture.cs